docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Controller Deployment
+sidebar_label: Controller
+id: deploy
+---
+
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md';
+
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide.
+
+## Installation
+
+The controller package `openziti-controller` provides a systemd service unit and optional bootstrapping. The package depends on the `openziti` package which provides the `ziti` CLI. The easiest way to install both is the cross-platform install script for RPM and Debian distributions.
+
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-controller
+```
+
+<LinuxPackageRepo />
+
+## Bootstrapping

```

Comment:
I'd like to rework this to be "Configuration" instead of "Bootstrapping".  It should offer two sections
* New Configuration
* Migrating an Existing Configuration

New Configuration is where you can introduce the newly generated "bootstrapping" process but I feel strongly that this should be a very explicit step that the user must complete. One of the lessons I've learned from the quickstarts is that people will get this wrong. Making it an explicit step, not implicitly accomplished, puts the onus onto the user to follow the instructions and not get it wrong.

At this point in the reading, it's not clear enough to me from reading the Bootstrapping section what will happen. I'd like to see the expectations/flow documented here and I'd like to steer people to the smallest set of steps we can and tell them how to do it with a "do this" sort of flow.

This would then also obviate the "Migrate" subpage on this node too. (maybe... depends on how it ends up)